### PR TITLE
I18n complement

### PR DESCRIPTION
**Pull request by biharck**
Adding missing i18n properties. This PR depends on react-viewerbase https://github.com/OHIF/react-viewerbase/pull/128

### CODE REVIEW

**Comment by biharck** [comment]
Changes done @swederik

File: extensions/ohif-i18n/src/locales/en-US/index.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ import CineDialog from './CineDialog.json';
 import Common from './Common.json';
 import Header from './Header.json';
 import MeasurementTable from './MeasurementTable.json';
-import UserPreferencesModal from './UserPreferencesModal.json';
```
Comment (swederik) [inline extensions/ohif-i18n/src/locales/en-US/index.js]: There shouldn't be any changes to the JSON or index.js files.

File: src/connectedComponents/ToolbarRow.js [hunk 2]
```diff
@@ -96,6 +97,32 @@ class ToolbarRow extends Component {
...
+        icon: 'list',
+        bottomLabel: t('Measurements'),
+      },
+    ];
+
+    const leftSidebarValue = this.props.leftSidebarOpen
+      ? leftSidebarToggle[0].value
+      : null;
+
+    const rightSidebarValue = this.props.rightSidebarOpen
```
Comment (swederik) [inline src/connectedComponents/ToolbarRow.js]: I don't see where these variables are being used at all?

File: extensions/ohif-i18n/src/locales/en-US/index.js (hunk 1 above)
Comment (biharck) [reply extensions/ohif-i18n/src/locales/en-US/index.js]: done

File: src/connectedComponents/ToolbarRow.js (hunk 2 above)
Comment (biharck) [reply src/connectedComponents/ToolbarRow.js]: It was after I merge with the latest master changes. I reverted already

**Pull request merged**